src/Moq/InvocationShape.cs

```diff
@@ -116,7 +116,7 @@ private bool IsOverride(MethodInfo invocationMethod)
 
 			if (method.IsGenericMethod)
 			{
-				if (!invocationMethod.GetGenericArguments().SequenceEqual(method.GetGenericArguments(), AssignmentCompatibilityTypeComparer.Instance))
+				if (!method.GetGenericArguments().CompareTo(invocationMethod.GetGenericArguments(), exact: false))
```

Comment:
Swapping the order of `method` and `invocationMethod` is no mistake; this swap previously happened in `AssignmentCompatibilityTypeComparer`, too (see below).